index.html

```diff
@@ -0,0 +1,234 @@
+										<li><a href="#"><i class="fab fa-twitter"></i></a></li>
+										<li><a href="#">English</a></li>
+										<li><a href="#">My Page</a></li>
+										<li><a href="#">Logout</a></li>
+								</ul>
+						</div>
+				</div>
+				<div class="container-nav">
+						<div class="logo-nav">
+								<a href="./index.html">
+										<img src="./assets/logo_ccglobal.png" alt="logo">
+								</a>
+						</div>
+						<ul class="header-nav-desktop">
+								<li><a href="./about.html">About</a></li>
+								<li><a href="#">Program</a></li>
+								<li><a href="#">Join</a></li>
+								<li><a href="#">Sponsor</a></li>
+								<li><a href="#">News</a></li>
+								<li><a href="#">CC Campaign</a></li>
+						</ul>
+				</div>
+		</nav>
+		<div class="container">
+				<div class="header-nav-phone">
+						<button type="button" class="menu-btn">
+								<img src="./assets/Union.svg" alt="menu link">
+						</button>
+						<nav class="header-nav-mobile hidden">
+								<button type="button" class="btn-close-nav">
+										<img src="./assets/Icon-Cancel.svg" alt="cancel icon">
+								</button>
+								<ul>
+										<li>
+												<a href="./about.html" class="link-about">
+														About
+												</a>
+										</li>
+										<li>
+												<a href="#" class="link-program">
+														Program
+												</a>
+										</li>
+										<li>
+												<a href="#" class="link-join">
+														Join
+												</a>
+										</li>
+										<li>
+												<a href="#" class="link-sponsor">
+														Sponsor
+												</a>
+										</li>
+										<li>
+												<a href="#" class="link-news">
+														News
+												</a>
+										</li>
+										<li>
+												<a href="#" class="link-cc-campaign">
+														CC Campaign
+												</a>
+										</li>
+								</ul>
+						</nav>
+				</div>
+				<div class="headline">
+						<h3>
+								"Hello! Techprenuers"
+						</h3>
+						<h1>
+							TECH STARTUPS GLOBAL SUMMIT 2015
+						</h1>
+						<p class="headline-paragraph-description">
+							Global Tech Startup Summit informs experts in the IT sector, managers,
+						  and start-ups on the newest technical advancements, allowing them to stay ahead of the curve.
+						</p>
+						<h2 class="headline-date">
+								2015.10.15(THU) ~ 16(FRI)
+						</h2>
+						<p class="headline-paragraph-location">
+								@National Musseum of Korea, Art Center Nabi and more
+						</p>
+				</div>
+		</div>
+</header>
+<main>
+  <div class="container">
+      <div class="main-title">
+          <p>Main Program</p>
+      </div>
+      <div class="main-program-container">
+          <div class="main-cards">
+              <div class="main-card">
+                  <div class="main-card-img">
+                      <i class="fas fa-book-reader"></i>
+                  </div>
+                  <div class="main-card-title">
```

Comment:
Please ensure that the HTML uses semantic tags, sections use `section`, articles use `article`, the nav and footer use `nav` and `footer` according to the project requirement.